Code/Source/solver/output.cpp

```diff
@@ -172,7 +172,7 @@ void read_restart_header(ComMod& com_mod, std::array<int,7>& tStamp, double& tim
 
 /// @brief Reproduces the Fortran 'WRITERESTART' subroutine.
 //
-void write_restart(Simulation* simulation, std::array<double,3>& timeP)
+void write_restart(Simulation* simulation, std::array<double,3>& timeP, Array<double>& An, Array<double>& Dn, Array<double>& Yn)
```

Comment:
Have a set order for all functions in which to pass the three variables. Either `A, Y, D` or `D, Y, A`.